### PR TITLE
fix: Added description about firebase configs

### PR DESCRIPTION
**Pull request by 0to1to0to1**

### CODE REVIEW

**Review by karolinayanko — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -34,25 +34,27 @@ Use root or package level commands to run specific package, e.g. web:
 
 [/packages/web] yarn start
 ```
+You should add your `google-services.json` from firebase configs to '[/packages/mobile/android/app].
 Use root or package level commands to run mobile:
 ```
-[/] yarn workspace mobile start:android
+[/] yarn workspace mobile android
```
Comment (dubbha) [inline README.md:44]: If you move away from the dev+android parallel set of commands, don't you need to compile first?

File: README.md [hunk 2]
```diff
@@ -34,25 +34,27 @@ Use root or package level commands to run specific package, e.g. web:
...
 
 [/packages/mobile] yarn start:android
```
Comment (dubbha) [inline README.md]: If you change the above command change this one too.

P.S. Both comments apply to the ios part below.

File: README.md (hunk 1 above)
Comment (artbitbyte) [reply README.md:44]: No, you don't

File: README.md (hunk 2 above)
Comment (artbitbyte) [reply README.md]: Thanks :)

**Review by dartwlad — APPROVED** [review]

**Pull request merged**